src/solver/type_getters.jl

```diff
@@ -787,6 +787,11 @@ function get_integrator(config::AtmosConfig)
     atmos = get_atmos(config, params)
     numerics = get_numerics(config.parsed_args)
     simulation = get_simulation(config, config.comms_ctx)
+    simulation.output_dir
```

Comment:
I think you might have forgot to remove this line

Reply:
Yes, this is a typo. thanks!